rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -258,35 +277,80 @@ namespace rsx
 		const u32 m_max_zombie_objects = 64; //Limit on how many texture objects to keep around for reuse after they are invalidated
 
 		//Other statistics
-		const u32 m_cache_miss_threshold = 8; // How many times an address can miss speculative writing before it is considered high priority
-		std::atomic<u32> m_num_flush_requests = { 0 };
-		std::atomic<u32> m_num_cache_misses = { 0 };
-		std::atomic<u32> m_num_cache_speculative_writes = { 0 };
-		std::atomic<u32> m_num_cache_mispredictions = { 0 };
+		std::atomic<u32> m_flushes_this_frame = { 0 };
```

Comment:
If we ever extend the lifetime of statistics beyond the scope of a single frame, we'll have to rename these again. Previous naming was more general imo. Just an observation.